index.html

```diff
@@ -29,13 +29,13 @@
     <nav class="full-menu">

```

Comment:
- [ ] Please descrease the size of `x` button and add padding on top as provided in Figma: 

Your design  | Figma design
------------- | -------------
<img width="363" alt="Screenshot 2022-04-05 at 13 10 52" src="https://user-images.githubusercontent.com/77038610/161720616-b119f37f-9844-40f4-b273-ac614782a006.png"> | <img width="363" alt="Screenshot 2022-04-05 at 13 08 45" src="https://user-images.githubusercontent.com/77038610/161720195-d398de0b-f7a7-4db1-8311-059bf1b20fff.png">